lib/presentation/screens/playlist_screen/bloc/traclist_bloc.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:spotify_prototype/data/repository/repository.dart';
+import 'package:spotify_prototype/presentation/screens/playlist_screen/bloc/tracklist_events.dart';
+import 'package:spotify_prototype/presentation/screens/playlist_screen/bloc/tracklist_states.dart';
+
+class PlayListBloc extends Bloc<TrackEvent, PlayListState> {
+  Repository repository;
+  PlayListBloc(this.repository) : super(InitialState()) {
+    on<SearchTrackEvent>((event, emit) async {
+      emit(LoadingTrack());
+      emit(LoadedTrack(await repository.fetchTrack(search: event.search, limit: event.limit)));
```

Comment:
handle errors here?

Reply:
Are you sure what I must do this?

Reply:
ofc u are. U have state, but u are not using it. And main reason - it is bad to show user loading indicator when u will not load anything coz of error.